packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -2010,3 +2011,1106 @@ class DraggableTable extends React.Component {
   }
 }
 ```
+
+### Composable: Sticky column
+
+To make a column sticky, wrap `TableComposable` with `InnerScrollContainer` and add the following properties to the `Th` that should be sticky: `isStickyColumn` and `hasRightBorder`. To prevent the default text wrapping behavior and allow horizontal scrolling, all `Th` should also have the `modifier="nowrap"` property. To set the minimum width of the sticky column, use the `stickyMinWidth` property.
+
+```js
+import React from 'react';
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  Caption,
+  TableText,
+  InnerScrollContainer
```

Comment:
can we add `InnerScrollContainer` to the props tables at the bottom?

Reply:
Added `OuterScrollContainer` as well